docs/.vuepress/config.js

```diff
@@ -294,6 +294,56 @@ module.exports = {
             },
           ],
         },
+        {
+          title: "IBC Light Client Developer Guide",
+          children: [
+            {
+              title: "Overview",
+              directory: false,
+              path: "/light-clients/overview.html",
+            },
+            {
+              title: "client_state.go and its functions",
```

Comment:
These will appear on the sidebar right? Would it look cleaner as just "ClientState"?
```suggestion
              title: "ClientState",
```